modules/gatsby-theme-ocular/CHANGELOG.md

```diff
@@ -1,10 +1,26 @@
 # CHANGELOG (gatsby-theme-ocular)
 
+# v1.1.0-alpha.3
+- Fixes support for multiple doc directories, removes need to add ignores to gatsby-config.js
+
+# v1.1.0-alpha.2
+Fixes
+
+# v1.1.0-alpha.1
+Refactor config handling:
+- Removes need for `ocular-config.js`: config goes into the theme in the normal `gatsby-config.js`.
+- Removes need for `gatsby-node.js` in most websites, meaning that most websites can just have a `gatsby-config.js`
+
+## v1.0.3 
+
+## v1.0.2 
+
 ## v1.0.1 
 - Fix npmjs.org READMEs
 
 ## v1.0.0 
-- First publish of ocular-gatsby repackaged as a theme
+First publish of ocular-gatsby repackaged as a theme
+- Removes need for `gatsby-browser.js` and `gatsby-ssr.js` in most websites.
 
 # CHANGELOG (ocular-gatsby)
```

Comment:
@jckr - I seemed to remember we kept a detailed CHANGELOG but can't find it now. Maybe that was just for the dev-tools?